app/grandchallenge/components/templates/components/partials/civ/image.html

```diff
@@ -9,10 +9,11 @@
         >
         <i class="fas fa-fw fa-download"></i>
         {% if display_inline %}
-            Download {{ civ.interface.title|truncatechars:64 }}:
+            Download {{ civ.interface.title|truncatechars:64 }}{% if civ.image.files.all|length > 1 %} {{ forloop.counter }}{% endif %}:
             {{ civ.image.name|name|truncatechars:30 }}
+            ({{ file.file|suffix|truncatechars:30 }})
         {% else %}
-            Download {{ civ.interface.title }}:
+            Download {{ civ.interface.title }}{% if civ.image.files.all|length > 1 %} {{ forloop.counter }}{% endif %}:
             {{ civ.image.name|stem}}
             ({{ file.file|suffix }})
         {% endif %}
```

Comment:
I don't think that the counter makes sense - it looks like this is producing multiple Generic Medical Image outputs. It's not, it's the same GMI just with two different representations. 

The titles we control, so they _should_ be short. The Download instruction seems unnecessary given the icon. I think using the stem consistently is better as the original suffix is irrelevant. Suffixes are always short. I suggest only varying the length of the file stem between inline display and not:

```
         {{ civ.interface.title }}
         {% if display_inline %}
             {{ civ.image.name|stem|truncatechars:16 }}
         {% else %}
             {{ civ.image.name|stem }}
         {% endif %}
         ({{ file.file|suffix }})
```